en/app.ftl

```diff
@@ -66,6 +66,40 @@ nav-footer-release-notes = Release Notes
 logo-mozilla-alt = { -brand-name-mozilla }
 logo-github-alt = GitHub logo
 
+## The new feature announcement area, under the "News" link at the top
+
+whatsnew-trigger-label = News
+whatsnew-counter-label = { $count } new announcements.
```

Comment:
Reconsider how this string should be coded because of numbers: https://firefox-source-docs.mozilla.org/l10n/fluent/review.html#variants-and-plurals

Reply:
Whoops, I knew that. Fixed in aa72508.